open-apparel-registry/open-apparel-registry
# Update parent company label

## Overview

Through the "Claim a Facility" mechanism, we request information on "Parent Company" which, if available on the OAR, is listed on claimed profiles. We heard during the recent OECD conference that many facility owners or factory groups do not understand use of the language "Parent Company" as it relates to company structure. This label has been updated to "Parent Company / Supplier Group" where it appears throughout the application.

Connects #959 

## Demo

<img width="724" alt="Screen Shot 2020-03-02 at 2 10 03 PM" src="https://user-images.githubusercontent.com/21046714/75708898-b1208a80-5c8f-11ea-824a-510260a02025.png">

## Testing Instructions

* Checkout this branch
* Run `vagrant ssh` and `./scripts/server`
* Claim a facility. When you get to Step 3 in the process, 'Parent Company' should read 'Parent Company / Supplier Group'
* Verify the facility claim through an admin account. 'Parent company' should be labeled 'Parent company / supplier group'
* As the user who claimed the facility, look at the facility details. 'Parent Company' should read 'Parent Company / Supplier Group'

## Checklist
